needy/projects/autotools.py

```diff
@@ -108,12 +108,13 @@ def configure(self, output_directory):
 
     def build(self, output_directory):
         make_args = make.get_make_jobs_args(self)
+        build_targets = self.build_targets()
```

Comment:
you're doing two different things here and in make. i'd say pick one (the more concise one imo)